### DISCUSSION TITLE
How to make Responsive desktop application using avalonia Framework which can run on different screen size device?

### DISCUSSION THREAD

**Discussion by cschandancs** [AvaloniaUI/Avalonia#7676] [Q&A]
I want to make responsive desktop application which can run on different screen size device .
I have used Grid.Extra.Avalonia library but did not get success. it is only working for border.
for eg. we want to make 2x2  column and row within that we can place another control like textbox ,label which can be responsive according to different scree size.
Any help appreciated.

**Comment by workgroupengineering** [comment]
see https://github.com/wieslawsoltes/AvaloniaBehaviors/blob/master/src/Avalonia.Xaml.Interactions/Responsive/AdaptiveBehavior.cs

**Reply by cschandancs** [reply]
can you provide sample example?
becasue i didn't get it.

**Reply by wieslawsoltes** [reply]
> can you provide sample example? becasue i didn't get it.

https://github.com/wieslawsoltes/AvaloniaBehaviors/blob/2f972ebb0066a2a4235126da7e103f684de1c777/samples/BehaviorsTestApplication.Base/Views/Pages/AdaptiveBehaviorView.axaml#L100-L139
